protocol-units/bridge/state_logic.md

```diff
@@ -1,67 +1,34 @@
 # Bridge transfer state logic definition
 
-## Normal transfer process state changes
+## Transfer process state changes
 
 ```mermaid
 flowchart TB
     L1{"L1"} --Event BridgeTransferInitiated--> B("Initialized")
-    B --Action lock_bridge_transfer--> L2("L2")
-    L2 --Event BridgeTransferLockedEvent--> C("Locked")
-    L2 --Event BridgeTransferCompletedEvent--> D("KnowSecret")
-    D -- Action BridgeTransferCompletedEvent --> L1
-    L1 -- Event BridgeTransferCompleted --> E("Done(Completed)")
-    L2 -- Event lock_bridge_transfer Failed --> F("NeedRefund")
+    B --Action complete_bridge_transfer--> L2("L2")
+    L2 --Event BridgeTransferCompletedEvent--> C("Completed")
+    L2 -- Event BridgeTransferCompleted Tx Failed --> D("Aborted")
```

Comment:
Why D("Aborted")?

Reply:
Aborted is a state. The line mean that a transfer has this state when a complete Tx fail.
An aborted transfer is replayed.